content/javascript/concepts/arrays/terms/includes/includes.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: '.includes()'
+Description: 'Returns true if a given value is included in an array.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.includes()` method returns `true` if a given value is included in an array. Otherwise, it returns `false`.
+
+## Syntax
+
+```pseudo
+array.includes(value, index=0);
+```
+
+The following parameters are used:
+
+- A case-sensitive and type-sensitive `value` that is checked for inclusion in the `array`.
+- An optional `index`, defaulted to 0, that tells `.includes()` where to begin the check.
+
+Here are some edge-cases to consider when using `.includes()`:
+
+- It will not work if the provided `index` is greater than the length of the array. Instead, `false` will be returned.
+- If the `index` is less than or equal to 0, the entire array will be searched.
+- By itself, this method isn't suitable for nested array and should be used with the `.flat()` and/or [`.find()`](https://www.codecademy.com/resources/docs/javascript/arrays/find) methods.
+- For objects, `.includes()` only returns `true` for references of the same object (even checking against an object with the same property-value pairs returns `false`).
+
+## Example
+
+The `.includes()` method can be used in a few ways. First, it can be used directly with an array:
+
+```js
+[1, 2, 3].includes(3);
```

Comment:
We should probably still include `console.log()` to accurately display an output ~

Reply:
Can we make this change for the rest of the examples as well? 🙏